resources/invoice.go

```diff
@@ -29,3 +29,33 @@ func (inv *Invoice) Fetch(invoiceID string, queryParams map[string]interface{},
 func (inv *Invoice) Create(data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
 	return inv.Request.Post(constants.INVOICE_URL, data, extraHeaders)
 }
+
+// Cancel cancels a invoice having the given invoiceID.
+func (s *Invoice) Cancel(invoiceID string, queryParams map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
+	url := fmt.Sprintf("%s/%s/cancel", constants.INVOICE_URL, invoiceID)
+	return s.Request.Post(url, queryParams, extraHeaders)
+}
+
+// Delete delete a draft invoice having the given invoiceID.
+func (s *Invoice) Delete(invoiceID string, queryParams map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
+	url := fmt.Sprintf("%s/%s", constants.INVOICE_URL, invoiceID)
+	return s.Request.Delete(url, queryParams, extraHeaders)
+}
+
+// Edit updates the invoice having the given invoiceID.
+func (s *Invoice) Edit(invoiceID string, data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
```

Comment:
```suggestion
func (s *Invoice) Update(invoiceID string, data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
```

Reply:
done even in unit test

Reply:
Rename [this](https://github.com/razorpay/razorpay-go/pull/54/commits/71a7bb52186fff13f33226d63a4432f087b4382c#diff-d8d87577c7ce940f6a5cfb10605997f4987df4ab761796b1a7af196d0d0285b5R89) to TestInvoiceUpdate